packages/dht/src/connection/WebSocket/WebSocketServer.ts

```diff
@@ -90,7 +90,7 @@ export class WebSocketServer extends EventEmitter<ConnectionSourceEvents> {
             })
 
             try {
-                this.httpServer.listen(port, host)
+                this.httpServer.listen(port, '0.0.0.0')
```

Comment:
Add a comment that listens all IPv4 interfaces, but not IPv6 interfaces (as it would do if we don't specify any host)